### PR TITLE
[TW-122] Mock by API refresh

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Renamed page to "Mocking with the Postman API"
- Rewrote all content
- Updated screenshots

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md [hunk 1]
```diff
@@ -35,63 +35,119 @@ contextual_links:
...
+
+> You can also find the mock server URL in Postman. Select **Mock Servers** in the left sidebar, select the `testAPI` mock server, and then select **Copy URL**.
+
+## Step 5: Send a request to the mock server
+
+You are now ready to simulate requests using your collection. To send a request to the mock server, use the mock server URL and append the request path: `{{mockURL}}/path`
+
+Try this yourself by simulating `Request 1` in the `testAPI` collection. Create a new request in Postman, leaving `GET` selected for the method. For the request URL, enter your mock server URL and append the path from the request:
+
+`https://5327d2ae-89ac-4568-b0cb-190b7f3408b2.mock.pstmn.io/get?test=123`
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md:127]: Same thought here - maybe use a placeholder value:

```suggestion
`https://<your-mock-server-url>/get?test=123`
```

**Review by HeatherStenson — APPROVED** [review]
Looks good! Just two suggestions, no blockers. 🍊

File: src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md [hunk 2]
```diff
@@ -35,63 +35,119 @@ contextual_links:
...
+Next, add the following **raw** JSON code to the **Body** tab of the request, substituting your collection ID and environment ID:
 
-```text
-https://{{mockId}}.mock.pstmn.io/{{mockPath}}
+```JSON
+{
+  "mock": {
+    "name": "testAPImock",
+    "collection": "16724969-97860d6e-742a-409d-ae1e-0d34a53c57d6",
+    "environment": "16724969-3a77ee2c-6155-4c2d-af9e-2bfc9dcc174f"
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md:100]: I think you're right--great suggestion! 💡

File: src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md:127]: Yep, great suggestion again

**Pull request merged**